cfme/tests/automate/test_common_methods.py

```diff
@@ -320,3 +321,60 @@ def _finalize():
         assert provision_request.is_succeeded(
             method="ui"
         ), f"Provisioning failed: {provision_request.row.last_message.text}"
+
+
+@pytest.fixture(scope="function")
```

Comment:
**Suggestion:**
@ganeshhubale Can you please move this fixture to above, not in between tests.